warehouse/schema.go

```diff
@@ -93,8 +96,8 @@ func BundleFields() []BundleField {
 	return result
 }
 
-// ExportTableSchema retrieves information abot the fields in the warehouse table into which data will 
-// finally be loaded.  
+// ExportTableSchema retrieves information abot the fields in the warehouse table into which data will
```

Comment:
Not your fault, but "abot" is misspelled here.

Reply:
done